hlx_statics/scripts/lib-helix.js

```diff
@@ -711,6 +712,45 @@ export function githubActionsBlock(doc) {
   }
 };
 
+/**
+ * parse attributes from row and added to particular blcok
+ */
+
+function parseAttribute(block){
+  const removeEmptyDivs = (element) => {
```

Comment:
this would remove any empty div from the block, and not just the ones that contained where the data-attributes came from